uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -498,7 +498,7 @@ define([
     function initRoles () {
       role = 'staff';
 
-      if (vm.isSelfRecord) {
+      if ($rootScope.section === 'my-leave') {
```

Comment:
is it possible to see my leave requests in a different section? can they be seen on manager leave? contact page?

Reply:
Yes, this component is used in Absence Tab:

![image](https://user-images.githubusercontent.com/3973243/29973951-329e6ab4-8f29-11e7-8f86-067a579e2159.png)